lib/grape-route-helpers/all_routes.rb

```diff
@@ -10,7 +10,13 @@ def decorated_routes
 
     def all_routes
       routes = subclasses.flat_map { |s| s.send(:prepare_routes) }
-      routes.uniq(&:options)
+      routes.uniq do |r|
```

Comment:
What do you think of implementing a method on the route that returns the attributes that make it unique? In this case it's the route options + the path. Something like this:

```ruby
class DecoratedRoute
  attr_reader :unique_key

  def initialize(route)
    # setting other instance variables
   @unique_key = route_options.merge(path: route.path)
  end
end
```
Then we can shorten lines 13-19 to `routes.uniq(&:unique_key)`. What do you think?